src/Event/EnhancedEcommerceEvents.php

```diff
@@ -20,8 +20,9 @@
   const ALTER_PRODUCT = 'commerce_google_tag_manager.alter_product';
 
   /**
-   * Allows to alter the event data of each Enhanced Ecommerce event
-   * before it gets pushed to the data layer.
+   * Allows to alter the event data of each Enhanced Ecommerce event.
+   *
+   * Allows alteration before it gets pushed to the data layer.
```

Comment:
Do you agree with this change ? 

The first comment must be a one line comment, the second can be multiline (drupal 8 coding standard).